### PR TITLE
add session1

### PR DESCRIPTION
**Pull request by Tahoora78**
add session1

### CODE REVIEW

File: session1(1401-05-2).md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+- head 	Displays first lines of a file
+- grep 	Prints lines that match patterns
+- wc 	Word count files
+	`cat test.txt | wc -l` --> number of lines in test.txt
+---
+### ways to see all of the commands that you have entered:
+- with up/down key
+- `history`
+- `cat ~/bash_history.sh`
+- ctrl + r ---> reverse searching smong entered commands
```
Comment (M-Heidari2000) [inline session1(1401-05-2).md]: 62 -> among

File: session1(1401-05-2).md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+-----
+
+## Books to read
+### python
+-   distilled david beazley
+-   fluent ramlho
+
+### Javascript
+-   you dont know js [Kyle Simpson]
+-   The difinnitive guide js david flanagan
```
Comment (M-Heidari2000) [inline session1(1401-05-2).md]: 154 -> definitive

File: session1(1401-05-2).md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+### git cheatsheet
+https://education.github.com/git-cheat-sheet-education.pdf
+
+### creating new branch
+```
+git checkout -b new_branch
+git push --set-upstream new_branch
+```
+### deletting a  branch
```
Comment (M-Heidari2000) [inline session1(1401-05-2).md]: 137 -> deleting

**Review by M-Heidari2000 — CHANGES_REQUESTED** [review]
Perfect! except for a couple of typos. I've added comments on those lines.

**Comment by Tahoora78** [comment]
I resolved the typos

**Review by lashkariam — APPROVED** [review]

**Review by M-Heidari2000 — APPROVED** [review]

**Review by amin-rami — APPROVED** [review]

**Review by amin-rami — APPROVED** [review]

**Pull request merged**